Documentation/ExtensionArchitecture/FileStructure/Resources/Public/Index.rst

```diff
@@ -39,18 +43,27 @@ By using the Composer package
 files except those that should be publicly available can be stored outside
```

Comment:
Since TYPO3 12 this is not needed anymore. Should we remove that part?

Reply:
Not really, you can still put files that should not be publicly available into the Ressources/Public folder. And in Classic Installation the issue prevails.